graviti/openapi/request.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+#
+# Copyright 2022 Graviti. Licensed under MIT License.
+#
+
+"""The basic concepts and methods of the Graviti OpenAPI."""
+
+from typing import Any
+from uuid import uuid4
+
+from requests.models import Response
+from tensorbay.exception import ResponseError, ResponseErrorDistributor
+from tensorbay.utility import config, get_session
+
+from graviti.__version__ import __version__
+
+URL_PATH_PREFIX = "api/v1"
+
+
+def open_api_do(url: str, access_key: str, method: str, **kwargs: Any) -> Response:
```

Comment:
The order is inconsistent with arguments.
The order should be: access_key, url, method, **kwargs

Reply:
I think `method` should be the first?

Reply:
Yes, then the order should be method, access_key, url, **kwargs